luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/camera_track_body/camera_clip_body.rs

```diff
@@ -60,11 +63,10 @@ impl CameraClipBody {
             builder.on_mouse_down(move |event| {
                 let event = EditorEvent::CameraClipBodyMouseDownEvent {
                     clip_id: clip_id.clone(),
-                    local_mouse_xy: event.local_xy,
-                    global_mouse_xy: event.global_xy,
+                    click_in_time: timeline_start_at + PixelSize(event.local_xy.x) * time_per_pixel,
```

Comment:
Previously I used mouse xy to change clip, but I think it's not a good idea, for example, if I change timeline.strat_at using wheel, the logic become sucks. To be clear, I changed it to use time instead of xy.